src/accelerate/components/App.css

```
@@ -20,3 +20,12 @@
 .App__blocks--three {
 	grid-template-columns: repeat( 3, 1fr );
 }
+
+.notices__wrap {
+	background-color: #f5f6f8;
+	overflow: hidden;
+}
+
+.notices__wrap > * {
+	margin: 15px 5% 0 5% !important;
```

Do we need `!important` here? It's best to avoid whenever possible. Can we rely on CSS specificity instead e.g. by adding another class name or ID to the rule like `#wpbody .notices__wrap > *`?

Also default notice styling appears to have a 5px bottom margin too:

```suggestion
	margin: 15px 5% 5px 5%;
```